baselines/Train.py

```diff
@@ -31,6 +34,13 @@ def train(epoch, trainloader, model, optimizer, criterion, args, device):
         inputs, targets = inputs.to(device), targets.to(device)
         scale_mean, scale_std = scale_mean.to(device), scale_std.to(device)
         output = model(inputs)
+
+        # special case for tformer
+        if args.model == 'Tformer':
+            # extract mean and stddev from ensemble and save it in output
+            output_mean, output_std = estimate_mean_std(output)
+            output = torch.cat((torch.movedim(output_mean, 0, -1), torch.movedim(output_std, 0,-1)), dim=-1)
+
```

Comment:
I think it's better to put this into forward() of Tformer.

Reply:
yes I agree, I will change that